docs/concepts/securing-lokomotive-cluster.md

```diff
@@ -0,0 +1,53 @@
+# Securing Lokomotive clusters
+
+Lokomotive clusters are deployed with security as a top concern.
+
+## Cluster wide Pod Security Policy
+
+A [Pod Security Policy](https://kubernetes.io/docs/concepts/policy/pod-security-policy/) is a
+cluster-level resource that controls security sensitive aspects of the pod specification. The
+PodSecurityPolicy objects define a set of conditions that a pod must run with in order to be
+accepted into the system, as well as defaults for the related fields.
+
+Lokomotive clusters have PodSecurityPolicy (PSP) enabled by default. The cluster comes with two
+default PSPs for general purpose application usage:
+
+  * [restricted](https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/psp-restricted.yaml)
+
+    Allowed to all the workloads in all namespaces except `kube-system` namespace. This PSP has the
+    following restrictions:
+
+    * Does not allow pods to be run as root.
+    * Allows only whitelisted volumes.
+    * Allows only whitelisted capabilities.
+    * The linux kernel host namespace sharing is not allowed for any pod using this PSP. The default
+      docker seccomp profiles are used.
+
+  * [privileged](https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/psp-privileged.yaml)
+
+    Allowed to workloads in `kube-system` namespace only. This PSP does not restrict any workloads
+    from any required permissions. Although this PSP can be used for a workload in another namespace
+    by creating a RoleBinding that binds the ClusterRole `privileged-psp` to corresponding
+    ServiceAccount, it is actually not recommended.
+
+    To allow special permissions to the workloads, the recommended way is to create a bespoke PSP and
```

Comment:
I think this should be right after describing the "restricted" policy and it should have a link to upstream documentation on how to do that.